pages/compare.py

```diff
@@ -42,6 +42,12 @@ def load_pat():
 
   with open(home + '/.clarifai_pat') as f:
     return f.read()
+  
+def show_error(request_name, response):
+  st.error("There was an error with your request to "+request_name)
+  st.json(json_format.MessageToJson(response, preserving_proto_field_name=True))
+  raise Exception("There was an error with your request to "+request_name+" "+response.status.description)
```

Comment:
better to use f-string formatted strings like: 
```
f"There was an error with your request to {request_name} {response.status.description}"
```
as it's less typing and easier to read :)

Reply:
done!